pkg/controller/controller.go

```diff
@@ -121,36 +121,38 @@ func (c *Controller) runPVCWorkers() {
 	}
 }
 
-// ProcessNextPodItem gets the next pod key from the queue and verifies that it was created
-// by the cdi controller. If not, the key is discarded. Otherwise, the pod object is passed
-// to processPodItem.
+// ProcessNextPodItem gets the next pod key from the queue and verifies that it was created by the
+// controller. If not the key is discarded; otherwise, the pod object is passed to processPodItem.
+// Note: pods are already filtered by label "app=containerized-data-importer".
 func (c *Controller) ProcessNextPodItem() bool {
 	key, shutdown := c.podQueue.Get()
 	if shutdown {
 		return false
 	}
 	defer c.podQueue.Done(key)
+
 	pod, err := c.podFromKey(key)
 	if err != nil {
-		c.forgetKey(key, fmt.Sprintf("Unable to get pod object: %v", err))
+		c.forgetKey(key, fmt.Sprintf("ProcessNextPodItem: unable to get pod from key %q: %v", key, err))
```

Comment:
One error case is that `key` is not parsable as a string, so `%q` will print out garbage.